sdk/lib/apisets/apisets.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * PROJECT:     ReactOS apisets
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Resolving the apiset to a ReactOS system dll
+ * COPYRIGHT:   Copyright 2024 Mark Jansen <mark.jansen@reactos.org>
+ */
+#include "apisetsp.h"
+
+
+const ULONGLONG API_ = (ULONGLONG)0x2D004900500041; /// L"API-"
+const ULONGLONG EXT_ = (ULONGLONG)0x2D005400580045; /// L"EXT-";
+
+WORD PrefixSize = sizeof(L"api-") - sizeof(WCHAR);
+WORD ExtensionSize = sizeof(L".dll") - sizeof(WCHAR);
+
+
+// The unittest wants to link against ntdll,
+// but the apisets library is used _inside_ ntdll so we have to make the symbol work in both cases.
```

Comment:
in the apitest it links against libntdll.lib for some other functions, and against unicode.obj via the lib `apisets`

Reply:
I still don't understand the problem. If you link to `_RtlCompareUnicodeString@xx` instead of `__imp_RtlCompareUnicodeString@xx` it should work in both cases. All you need to do is define `_NTSYSTEM_`

Reply:
That works, nice.